common/OpTestHMC.py

```diff
@@ -481,6 +481,25 @@ def get_lmb_size(self):
         '''
         return self.run_command("lshwres -r mem -m %s --level sys -F mem_region_size" % self.mg_system)
 
+    def configure_16gb_hugepage(self, num_hugepages):
+        '''
+        Configures managed system with 16gb hugepage size passed as argument
+
+        :param hgpg_size: 16gb hugepage size to configure on managed system
+        '''
+        self.run_command("chhwres -m %s -r mem -o s -a requested_num_sys_huge_pages=%s" %
+                               (self.mg_system, num_hugepages))
+        time.sleep(2)
+
+    def get_16gb_hugepage_size(self):
+        '''
+        Get current 16gb hugepage size
```

Comment:
same here do not hardcode 16gb

Reply:
the above two functions we use in order to know whether the system has configured with 16gb .
If not it is going to configure number of 16gb hugepages as per the user given value in machine config

Reply:
but keep it generic like get_hugepage_size

Reply:
okay